visual-studio-dev-vm/azuredeploy.parameters.json

```diff
@@ -1,33 +1,33 @@
-{
-  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
-  "contentVersion": "1.0.0.1",
-  "parameters": {
-    "storageName": {
-      "value": "myuniquestoragename"
-    },
-    "vmIPPublicDnsName": {
-      "value": "myvmuniqueip"
-    },
-    "storageType": {
-      "value": "Premium_LRS"
-    },
-    "deployLocation": {
-      "value": "East US 2"
-    },
-    "vmName": {
-      "value": "VSVM"
-    },
-    "vmAdminUserName": {
-      "value": "YourAdminName"
-    },
-    "vmAdminPassword": {
-      "value": "YourAdminPassw0rd!"
-    },
-    "vmSize": {
-      "value": "Standard_DS2"
-    },
-    "vmVisualStudioVersion": {
-      "value": "VS-2015-Enterprise-AzureSDK-2.7-WS2012R2"
-    }
-  }
-}
+{  
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+   "contentVersion":"1.0.0.1",
+   "parameters":{  
+      "storageAccountNamePrefix":{  
+         "value":"GEN_UNIQUE_11"
+      },
+      "vmIPPublicDnsName":{  
+         "value":"GEN-UNIQUE-13"
```

Comment:
@sedouard:
Thanks. Your comment led me to the solution: Using dashes instead of underscores.